attributes/default.rb

```diff
@@ -521,6 +521,9 @@
 
 default['hops']['nvidia_pkgs']['download_url']        ="#{node['download_url']}/kube/nvidia"
 
+default['hops']['docker']['git']['image']['name'] = "git"
```

Comment:
You should add a version for this git image
default['hops']['docker']['git']['image']['version'] = "0.1"